src/components/staking/PageValidator.vue

```diff
@@ -145,25 +139,23 @@
           <div class="column">
             <dl class="info_dl">
               <dt>Current Commission Rate</dt>
-              <dd>{{ num.percent(validator.commission.rate) }}</dd>
+              <dd>{{ percent(validator.commission.rate) }}</dd>
             </dl>
             <dl class="info_dl">
               <dt>Max Commission Rate</dt>
-              <dd>{{ num.percent(validator.commission.max_rate) }}</dd>
+              <dd>{{ percent(validator.commission.max_rate) }}</dd>
             </dl>
             <dl class="info_dl">
               <dt>Max Daily Commission Change</dt>
-              <dd>{{ num.percent(validator.commission.max_change_rate) }}</dd>
+              <dd>{{ percent(validator.commission.max_change_rate) }}</dd>
             </dl>
             <dl class="info_dl">
               <dt>Last Commission Change</dt>
               <dd>{{ lastCommissionChange }}</dd>
             </dl>
             <dl class="info_dl">
               <dt>Self Delegation</dt>
-              <dd id="page-profile__self-bond">
-                {{ selfBond }}
-              </dd>
+              <dd id="page-profile__self-bond">{{ selfBond }}</dd>
```

Comment:
Note: Somehow formatting flip flops. Let's watch this and fix it if it stays this way.

Reply:
I know, it's gone nuts. I use the cli to perform final linting/fixing now since it doesn't like my VSCode version. There are some conflicting rules somewhere.